README.md

```diff
@@ -4,3 +4,26 @@
 [![npm version](https://badge.fury.io/js/eslint-config-amo.svg)](https://badge.fury.io/js/eslint-config-amo)
 
 Shared eslint config for all amo related JS projects.
+
+## Versioning
+
+This project follows the [semantic versioning](https://semver.org/) specification.
+
+In order to release a new version, please follow these steps:
+
+1. make sure your local `master` branch is up-to-date
```

Comment:
Nit: Each of these items should probably start with a capital letter and end with a period.